build/release.mk

```diff
@@ -17,6 +24,13 @@ else
 	mkdir -p $(GOBIN)/darwin_amd64
 	env GOOS=darwin GOARCH=amd64 $(GO) build -o $(GOBIN)/darwin_amd64 $(GOFLAGS) -trimpath -ldflags '$(LDFLAGS)' ./...
 endif
+	@echo Build OSX arm64
+ifeq ($(BUILDER_GOOS_GOARCH),"darwin_arm64")
```

Comment:
Since these will be part of the official releases, I am wondering of the confidence we have in darwin/arm64?

Totally fine with linux/arm64, but maybe we want to add somewhere that linux/amd64 is our primary target and fully tested. Otherwise it might start giving incorrect expectations.

cc @jwilander

Reply:
we currently don't test the windows and mac 

but we also starting to have some developers using the Mac m1 so might be good to have this